ComposureWatch/src/Components/Card/CreateCard.jsx

```diff
@@ -35,6 +35,14 @@ const CreateCard = () => {
     setCardVar(currentCards);
   };
 
+  const cardRemoveChangeHandler = (enteredId) => {
```

Comment:
Could you come up with a better name for the `enteredId`? Maybe `cardId` would better describe what it is?